Source/IntegrationTests/TestFiles/LitTests/LitTest/ast/const.dfy

```diff
@@ -0,0 +1,5 @@
+// RUN: %testDafnyForEachResolver "%s" -- --allow-axioms=false
```

Comment:
I'm a bit puzzled, how is there --allow-axioms=false (which I did not know of before today), an {:axiom} in this file (so something seemingly not allowed), and no error or warning in the expect file?

Reply:
`--allow-axioms=false` is the default actually. If you use `--allow-axioms=true` then no warnings are generated even if you leave out `{:axiom}`.

Maybe the option should be renamed to `--allow-implicit-axioms`

Reply:
Thanks for the clarification. Indeed, I see how it would be worth it renaming this option --assume-are-axioms or something like that, but that'll be a separate discussion.